src/cli/commands/test/tests.rs

```diff
@@ -1,12 +1,27 @@
 use crate::cli::commands::{test::TestArgs, CommandExecution};
+use cairo_rs::serde::deserialize_program::deserialize_program_json;
+use colored::Colorize;
 use std::path::PathBuf;
 
 use super::{compile_and_list_entrypoints, setup_hint_processor, test_single_entrypoint};
 
 pub fn run_single_test(test_name: &str, test_path: &PathBuf) -> (String, bool) {
 	let (_, path_to_compiled, _) = compile_and_list_entrypoints(test_path.to_owned()).unwrap();
+
+	let program_json = match deserialize_program_json(&path_to_compiled) {
+		Ok(program_json) => program_json,
+		Err(e) => {
```

Comment:
It would make sense to return an actual error instead of a tuple (message, status), but a lot of functions that. So yeah, I think it might be better to open an issue to change the errors management everywhere in the code